TESTS/mbed_drivers/watchdog/main.cpp

```diff
@@ -18,6 +18,10 @@
 #error [NOT_SUPPORTED] Watchdog not supported for this target
 #else
 
+#ifndef MAX_IWDG_PR
+#define MAX_IWDG_PR 0x6
```

Comment:
Why is the max pre-scaler defined as 6 ?

Reply:
Right, I didn't look to other targets than STM and assumed that it is used like this in all boards. I'll take a peek to other drivers and make it work on all devices.